### PR TITLE
setup

### PR DESCRIPTION
**Pull request by Alfredbis29**
**In this Milestone of my project I:** 

- Create a new app. Give it a meaningful name.
- Initialize your project with Git.
- Make sure that your project has Postgres database set up.
- Build your project schema.
- Create and run the necessary migration files.
- Table and column names should match the ERD diagram.
- note: photo for users table should be a link to an image
- Foreign keys should be included.
- All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/migrate/20220713140900_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :bio
+      t.string :photo
+      t.string :posts_counter
```
Comment (mikethreels) [inline db/migrate/20220713140900_create_users.rb:7]: - [x] the post counter is supposed to be an integer.

File: db/migrate/20220713141153_create_post.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Title
+      t.string :Text
+
+      t.timestamps
+    end
```
Comment (mikethreels) [inline db/migrate/20220713141153_create_post.rb:8]: - [x] in the posts table there should be a comments counter and likes counter as well.

File: db/migrate/20220713141028_create_comment.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :Text
+
+      t.timestamps
```
Comment (mikethreels) [inline db/migrate/20220713141028_create_comment.rb:6]: - [x] the comments table should have a reference to the author and the posts.

File: db/migrate/20220713141250_create_like.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
+    end
```
Comment (mikethreels) [inline db/migrate/20220713141250_create_like.rb:6]: - [x] in the likes table the should be a reference to the author and the posts.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Alfred, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Alfredbis29 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220713141250_create_like.rb (hunk 4 above)
Comment (Alfredbis29) [reply db/migrate/20220713141250_create_like.rb:6]: Hello @mikethreels , Thanks for the professional review I follow all requirements and I changed it as mentioned . Thanks you can now check

**Pull request merged**